pyart/graph/radardisplay.py

```diff
@@ -708,7 +774,19 @@ def plot_colorbar(self, mappable=None, field=None, label=None, cax=None,
             if field is None:
                 field = self.plot_vars[-1]
             label = self._get_colorbar_label(field)
-        cb = fig.colorbar(mappable, ax=ax, cax=cax)
+
+        if orient is None:
+            orient = 'vertical'
+
+        # Find the axes locations to set colorbar
+        if cax is None:
```

Comment:
What is the benefit of using `make_axes_locatable` and having fixed padding and sizes for the colorbar as opposed to having matplotlib colorbar function do the resizing?